kodilla-spring-basic/src/main/java/com/kodilla/spring/basic/spring_configuration/homework/Cabrio.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.spring.basic.spring_configuration.homework;
+
+import org.springframework.stereotype.Component;
+import java.time.LocalTime;
+
+@Component
+public class Cabrio implements Car{
+
+    public boolean hasHeadlightsTurnedOn(LocalTime hour) {
+        return !(!hour.isAfter(LocalTime.of(20, 00)) && !(hour.isBefore(LocalTime.of(6, 00))));
```

Comment:
ten warunek mozna uproscic :)
Zreszta ten warunek powinien byc taki (co do zgodnosci argumentow)
return hour.isAfter(20:00) && hour.isBefore(6:00)

Reply:
Jak jest duzo negacji ! to ciezko sie czyta taki warunek